src/lib/utils/testUtils.ts

```diff
@@ -157,3 +159,16 @@ export const dropTestDatabase = async (): Promise<void> => {
   });
   await adminKnex.destroy();
 };
+
+export const startRedisConnection = async (): Promise<void> => {
+  try {
+    await connect();
+  } catch (error) {
+    log.error('Error connecting redis!', error);
+    process.exit(1);
+  }
+};
+
+export const stopRedisConnection = async (): Promise<void> => {
+  await disconnect();
+};
```

Comment:
Teetkö funkkarin kanssa jolla voi nopsaan flushata rediksen esim testien välissä?

Reply:
Testien toimivuuden kannalta ei pitäisi olla tarvetta tyhjennellä redistä. Tuo käyttää kuitenkin samaa Redis-intanssia kuin normi deviympäristö niin se tyhjennys voi häiritä devausta

Reply:
Miksi se häiritsisi devausta? 

Jokaisen testin olisi hyvä aloittaa puhtaalta pöydältä. Jos joskus aletaan kakuttamaan esim vastauksia, niin olis kiva että kun testi a) kakuttaa sen vastauksen niin testi b) ei ime sitä vaan suoraan kakusta vaan oikeasti käväisee siellä tietokannassa

Reply:
Tää pitäis tehä jotenkin niin, että olisi erillinen tyhjänä käynnistetty redis testejä varten.

Reply:
https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/issues/80